packages/widget.api/src/config.ts

```diff
@@ -26,9 +26,21 @@ interface ArrayParameterDefinition<S extends ParameterValue.PrimitiveType> {
   displayName: string;
   subType: S;
 }
+interface ObjectSetParameterDefinition<T extends ObjectType> {
+  type: "objectSet";
+  displayName: string;
+  objectType: T;
+}
 export type ParameterDefinition =
   | PrimitiveParameterDefinition<ParameterValue.PrimitiveType>
-  | ArrayParameterDefinition<ParameterValue.PrimitiveType>;
+  | ArrayParameterDefinition<ParameterValue.PrimitiveType>
+  | ObjectSetParameterDefinition<ObjectType>;
+
+export type ManifestParameterDefinition =
```

Comment:
It wouldn't directly be mapped from ParameterDefinition but do you think it's clearer to follow if we just redefined the types:
```
interface ManifestObjectSetParameterDefinition<T extends ObjectType> {
  type: ObjectSetParameterDefinition<T>["type"];
  displayName: string;
  objectTypeRids: [string];
}
export type ManifestParameterDefinition =
  | PrimitiveParameterDefinition<ParameterValue.PrimitiveType>
  | ArrayParameterDefinition<ParameterValue.PrimitiveType>
  | ManifestObjectSetParameterDefinition<ObjectType>;
```